proto/heartbeat.go

```diff
@@ -23,6 +23,13 @@ import (
 	"time"
 )
 
+// TODO(marc): we should assert
+// var _ security.AnyUserRequest = &PingRequest{}
+// here, but we need to break cycles first.
+
+// AnyUser implements security.AnyUserRequest.
```

Comment:
The problem is that the tests you asked me revive depend on this, because they are using root certs with requests that are hardcoded to the node user. Which would you rather sacrifice?